### PR TITLE
Add family codes

### PR DESCRIPTION
**Pull request by JoeTonDev**
Fix family codes API

### CODE REVIEW

File: src/accounts.rs [hunk 1]
```diff
@@ -2,24 +2,49 @@ use log::error;
...
+use crate::messages::{IncomingMessages, self};
 use crate::{server_versions, Client, Error};
 
 mod decoders;
 mod encoders;
 
 #[derive(Debug, Default)]
 pub struct Position {
-    /// Account holding position
+    // Account holding position
```
Comment (wboayue) [inline src/accounts.rs:13]: The `///` includes adds the field description to the generated documentation.

File: src/accounts.rs [hunk 2]
```diff
@@ -2,24 +2,49 @@ use log::error;
...
+    // Size of position
     pub position: f64,
-    /// Average cost of position
+    // Average cost of position
     pub average_cost: f64,
 }
 
+#[derive(Debug, Default)]
+pub struct FamilyCode {
+    // Acount ID
```
Comment (wboayue) [inline src/accounts.rs:25]: use `///`

File: src/accounts.rs [hunk 3]
```diff
@@ -2,24 +2,49 @@ use log::error;
...
+    pub account_id: String,
+    // Family code
+    pub family_code: String,
+}
+
+#[derive(Debug, Default)]
+pub struct PositionMulti {
+    // Request ID
```
Comment (wboayue) [inline src/accounts.rs:33]: use `///`

File: src/accounts.rs [hunk 4]
```diff
@@ -42,7 +67,22 @@ pub(crate) fn cancel_positions(client: &Client) -> Result<(), Error> {
     Ok(())
 }
 
+
+
+
+
+// Determine whether an account exists under an account family and find the account family code.
+pub(crate) fn family_codes(client: &Client) -> Result<impl Iterator<Item = FamilyCode> + '_, Error> {
+    client.check_server_version(server_versions::ACCOUNT_SUMMARY, "It does not support fammily code requests.")?;
```
Comment (wboayue) [inline src/accounts.rs:76]: typo `fammily` -> `family`

**Comment by JoeTonDev** [comment]
Hi Wil, 

Fixed typo and three slashes added.

**Comment by JoeTonDev** [comment]
Hi Wil,  Please delete this pull request.  I am going to start over.  Sorry, I've made a mess.

**Pull request closed without merging**